index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Helen Abera's Portfolio</title>
+</head>
+<body>
+
+    <h1>Helen Abera</h1>
+
+    <section id="About">
+        <h2>About</h2>
+            <p>My name is Helen and I am currently on my journey to becoming a software developer. I am excited to be learning through Code The Dream!</p>
+    </section>
+
+    <section id="Experience">
+        <h2>Experience</h2>
+            <ul>
+                <li>Warner Bros Entertainment</li>
+                    <img src="wb_photo.jpeg" alt="Warner Bros Studios Logo">
+                <li>University of Washington</li>
+                    <img src="UW.jpeg" alt="University of Washington Campus">
```

Comment:
Just a small thing here. `<li>` should really be the only type of element that is a direct child of `<ul>` to be semantically correct HTML.  The `<img>` can just go inside the `<li>` and then can be styled however you want